brigadier/projects/admin.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib import admin
+from django.utils.translation import gettext_lazy as _
+
+from .models import Project, Task
+
+
+class TaskInLine(admin.StackedInline):
```

Comment:
it's good to have tasks inline,
but it also need to create a standalone tasks admin entity